Formula/zrepl.rb

```diff
@@ -0,0 +1,72 @@
+class Zrepl < Formula
+  desc "One-stop ZFS backup & replication solution"
+  homepage "https://zrepl.github.io"
+  url "https://github.com/zrepl/zrepl/archive/v0.1.1.tar.gz"
+  sha256 "0c16554e4527d14a390d78cf95bce759da425019a83ec63acfed5b4c50d68c9c"
+  head "https://github.com/zrepl/zrepl.git"
+
+  depends_on "go" => :build
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/zrepl/zrepl").install contents
+
+    ENV["GOPATH"] = gopath
+    ENV.prepend_create_path "PATH", gopath/"bin"    
+    cd gopath/"src/github.com/zrepl/zrepl" do
+      system "./lazy.sh",  "godep"
```

Comment:
How is this handling dependencies?

Reply:
it will fetch the go dependencies temporarily for the build as detailed here:
https://github.com/zrepl/zrepl/blob/master/lazy.sh

I think this one will use these:
https://github.com/zrepl/zrepl/blob/master/Gopkg.lock
https://github.com/zrepl/zrepl/blob/master/Gopkg.toml

Reply:
Versions are pinned, so that's good. But is there a way not to download `dep` and use Homebrew's?

Reply:
I guess, but then I'd have to keep the deps with versions up to date in the homebrew formula so I don't really see much value added.

Reply:
I think you misunderstood the question. The question is if you could use the `dep` tool without manually downloading it in `./lazy.sh`. Homebrew ships with a version so if you only download conditionally (`if ! command -v dep; then`) we don't have a random unversioned download of a tool homebrew provides.

Reply:
sry for the late reply, I was on holiday. Now I understand better and will have a look

Reply:
wouldn't this mean I'd need to have changes to lazy.sh merged upstream?

Reply:
Yes it does. But using the current approach it's simply not acceptable in homebrew-core.

Reply:
just pushed a new commit to my branch that should solve this